extensions/wikia/AssetsManager/config.php

```diff
@@ -2448,7 +2448,8 @@
 	'type' => AssetsManager::TYPE_SCSS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
-		'//extensions/wikia/PortableInfobox/styles/PortableInfobox.scss'
+		'//extensions/wikia/PortableInfobox/styles/PortableInfobox.scss',
+		'//extensions/wikia/PortableInfobox/styles/PortableInfoboxEuropaTheme.scss'
```

Comment:
Can we load them based on the current theme instead of always loading all themes CSS?